packages/core/core/src/Parcel.js

```diff
@@ -257,3 +261,7 @@ export default class Parcel {
     return Promise.all(promises);
   }
 }
+
+export {default as Asset} from './Asset';
+export {default as Dependency} from './Dependency';
+export {default as Environment} from './Environment';
```

Comment:
Probably should make an index file to export all of these and Parcel but ok for now